hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/regionserver/BlockCacheTmpl.jamon

```diff
@@ -348,7 +363,19 @@ are combined counts. Request count is sum of hits and misses.</p>
         <td>Size of DATA Blocks</td>
     </tr>
 </%if> 
-<& evictions_tmpl; bc = bc; &>
+<%if lru %>
+    <tr>
+        <td>Size of Index Blocks</td>
+        <td><% String.format("%,d", ((LruBlockCache)bc).getCurrentIndexSize()) %></td>
```

Comment:
I think the only improvement that needs be done here is these should also be formatted with TraditionalBinaryPrefix.long2String().